pontoon/api/urls.py

```diff
@@ -41,7 +41,12 @@
     path(
         "entities/<int:pk>/",
         views.EntityIndividualView.as_view(),
-        name="entity-individual",
+        name="entity-individual-id",
+    ),
+    path(
+        "entities/<slug:project_slug>/<path:resource>/<str:key>/",
```

Comment:
For consistency, this should be either:
* `entities/<slug:project>/<path:resource>/<str:entity>/`
* `entities/<slug:slug>/<path:path>/<str:key>/`
* `entities/<slug:project_slug>/<path:resource_path>/<str:entity_key>/`